src/zenml/analytics/context.py

```diff
@@ -211,15 +211,10 @@ def group(
         """
         success = False
         if self.analytics_opt_in and self.user_id is not None:
-            if traits is None:
-                traits = {}
-
-            traits.update({"group_id": group_id})
```

Comment:
Just out of curiosity, why did we remove this here?

Reply:
This was (I guess) intended for mixpanel at some point. However it never worked, as that `group_id` keyword was never included in the `track` events. This is why I switched this to `server_id` (added as a trait when calling this function) as we also send the `server_id` with all our track events.